build-logic/src/main/kotlin/org/jetbrains/conventions/maven-publish.gradle.kts

```
@@ -56,3 +58,9 @@ publishing {
         }
     }
 }
+
+plugins.withType<ShadowPlugin>().configureEach {
+    // manually disable publication of Shadow elements https://github.com/johnrengelman/shadow/issues/651#issue-839148311
```

Can we leave some more information for not only what it does, but why this is needed? So that the future maintainers are not afraid to remove/change it later on if they need to

```suggestion
    // manually disable publication of Shadow elements https://github.com/johnrengelman/shadow/issues/651#issue-839148311
    // This is done to preserve compatibility and have the same behaviour as previous versions of Dokka.
    // For more details, see https://github.com/Kotlin/dokka/pull/2704#issuecomment-1499517930
```